src/Aggregation/GeotileGrid.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Elastica\Aggregation;
+
+/**
+ * Class GeotileGrid.
+ *
+ * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-geotilegrid-aggregation.html
+ */
+class GeotileGrid extends AbstractAggregation
```

Comment:
We should probably start marking these kind of class as `final` as @thePanz recently commented.
And as it's a new class also suffix the its name with `Aggregation`. This requires #1887 to be merged before.

Reply:
ok, no problem

Reply:
ping @ruflin for his confirmation.

Reply:
@ruflin  is this ok for you? prefer confirming before doing this changes

Reply:
@deguif can you point me to the final comment? I'm not sure I fully follow why we should make it final? Should we really prevent others from extending these classes?

Reply:
Let's not mark the class as final for now as it would need discussing this subject more in details.
Will the name `GeotileGridAggregation` be OK for you?

Reply:
@deguif Currently we always skip the Aggregation part as it is already in the name of the directory. So to be consistent it should be `GoetileGrid`?

Reply:
Yes but as stated in #1824 I would be in favor to deprecate all aggregations classes and suffix them with `Aggregation` to avoid them colliding with reserved keywords and unifying all classes names. 

Here's the class is new so it could directly be named accordingly and avoid having to deprecate it later.
Would that sound good for you?

Reply:
@ruflin  @deguif please align about it. I really would like it to be merged soon - one way or another ;) 

I have added this `Aggregation` at the end per request here. I have also changed code in my project to support this new name already.

Btw, I personally like adding Aggregation to the name.  Easlier my projects code looked like this  

`use Elastica\Aggregation\GeotileGrid as GeotileGridAggregation;`

for readibility reason later in code. Otherwise it is much less visible in code that this is aggregation.